armi/reactor/tests/test_flags.py

```diff
@@ -53,9 +58,17 @@ def test_lookup(self):
         )
         # order in CONVERSIONS can matter for multi word flags.
         # tests that order is good.
-        for conv, flag in flags.CONVERSIONS.items():
-            if len(conv.split()) > 1:
-                self.assertEqual(flags.Flags.fromString(conv), flag)
+        for conv, flag in flags._CONVERSIONS.items():
+            # the conversions are specified as RE patterns, so we need to do a little
+            # work to get them into something that can serve as candidate input. Since
+            # we are only using \b and \s+, this is pretty straightforward. If any more
+            # complicated patterns work their way in there, this will need to become
+            # more sophisticated. One might be tempted to bake the plain-text versions
+            # of the conversions in the collection in the flags module, but this is
+            # pretty much only needed for testing, so that wouldn't be appropriate.
```

Comment:
Just so I understand, you're wanting to take the conversions and recover the possible user input that might lead to those conversions for test coverage?

Reply:
Yeah, just taking the regular expression string and converting it to something it would `match`